torch/ao/quantization/_dbr/quantization_state.py

```diff
@@ -470,12 +469,7 @@ def get_op_convert_info(
             output_tensor_infos = seen_op_info.output_tensor_infos
             tensor_id = output_tensor_infos[0].id
             scale, zp = self.tensor_id_to_scale_zp[str(tensor_id)]
-            # TODO(future PR): remove this boolean flag
-            if not unwrap_scale_zp:
-                additional_kwargs.update({'scale': scale, 'zero_point': zp})
-            else:
-                additional_kwargs.update(
-                    {'scale': scale.item(), 'zero_point': zp.item()})  # type: ignore[dict-item]
+            additional_kwargs.update({'scale': scale, 'zero_point': zp})
```

Comment:
why not just always call `.item` here?

Reply:
doing that leads to a runtime error, I did not debug it that closely yet.  Can follow-up if needed.